cmd/streaming-v4-unsigned.go

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) 2015-2022 MinIO, Inc.
```

Comment:
```suggestion
// Copyright (c) 2015-2023 MinIO, Inc.
```

Reply:
Hey, it was correct when I started the PR :D

I will do a final pass before I mark it ready. I want PutObject to also work before that, though.